Berksfile

```diff
@@ -6,6 +6,8 @@ source 'https://supermarket.chef.io'
 
 metadata
 
+cookbook 'languages', git: 'https://github.com/chef-cookbooks/languages.git', branch: 'ruby_install'
```

Comment:
I'll get `languages` publishing to supermarket.chef.io. That will ensure we don't have to cargo cult `Berksfile` entries all over the place (including Delivery build cookbooks).